convert2rhel/unit_tests/actions/system_checks/eus_test.py

```diff
@@ -0,0 +1,63 @@
+# Copyright(C) 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+import datetime
+
+import pytest
+
+from convert2rhel import actions, systeminfo
+from convert2rhel.actions.system_checks import eus
+from convert2rhel.systeminfo import Version, system_info
+from convert2rhel.unit_tests.conftest import centos8
+
+
+@pytest.fixture
+def eus_action():
+    return eus.EusSystemCheck()
+
+
+class DateMock(datetime.date):
+    @classmethod
+    def today(cls):
+        return cls(2023, 11, 15)
+
+
+class TestEus:
+    @centos8
+    def test_eus_warning_message(self, eus_action, monkeypatch, global_tool_opts, pretend_os):
+
+        version = Version(8, 8)
```

Comment:
I think we should transform this into a pytest.parametrize to cover that 8.8, 8.10 (and 9.2?) will be covered by the test… 

Would be good to have a case where the version is set to 8.6 too, just to verify that this action is not doing something incorrectly.

Reply:
Sounds good